src/python_testing/TC_DRLK_2_9.py

```diff
@@ -0,0 +1,566 @@
+        try:
+            response = await self.send_single_cmd(cmd=drlkcluster.Commands.GetUser(userIndex=userindex),
+                                                  endpoint=self.app_cluster_endpoint,
+                                                  timedRequestTimeoutMs=1000)
+            asserts.assert_true(type_matches(response, drlkcluster.Commands.GetUserResponse),
+                                "Unexpected return type for GetUserResponse")
+            asserts.assert_true(response.userIndex == userindex,
+                                "Error when executing GetUserResponse command, credentialExists={}".format(
+                                    str(response.userIndex)))
+            asserts.assert_true(response.userName == username,
+                                "Error when executing GetUserResponse command, credentialExists={}".format(
+                                    str(response.userName)))
+            asserts.assert_true(response.userUniqueID == useruniqueid,
+                                "Error when executing GetUserResponse command, credentialExists={}".format(
+                                    str(response.userUniqueID)))
+            asserts.assert_true(response.credentials == credentiallist,
+                                "Error when executing GetUserResponse command, credentialExists={}".format(
+                                    str(response.credentials)))
+            asserts.assert_equal(expected_status, Status.Success)
+
+        except InteractionModelError as e:
+            asserts.assert_equal(e.status, expected_status, f"Unexpected error returned: {e}")
+
+    async def get_credentials_status(self, credentialIndex: int, credentialType: drlkcluster.Enums.CredentialTypeEnum,
+                                     step, userIndex, credential_exists):
+        if step:
+            self.step(step)
+        try:
+            credentials_struct = drlkcluster.Structs.CredentialStruct(credentialIndex=credentialIndex,
+                                                                      credentialType=credentialType)
+            response = await self.send_single_cmd(endpoint=self.app_cluster_endpoint, timedRequestTimeoutMs=1000,
+                                                  cmd=drlkcluster.Commands.GetCredentialStatus(
+                                                      credential=credentials_struct))
+            asserts.assert_true(type_matches(response, drlkcluster.Commands.GetCredentialStatusResponse),
+                                "Unexpected return type for GetCredentialStatus")
+            asserts.assert_true(response.credentialExists == credential_exists,
+                                "Error when executing GetCredentialStatus command, credentialExists={}".format(
+                                    str(response.credentialExists)))
+            if (not credential_exists):
+                asserts.assert_true(response.userIndex == NullValue,
```

Comment:
replace credentialExists with userIndex as that is the value used in format function